composer.json

```diff
@@ -39,7 +39,6 @@
         "webmozart/assert": "^1.11"
     },
     "require-dev": {
-        "brianium/paratest": "^6.7",
```

Comment:
7.0 was released yesterday and support PHPUnit 10 now 👍

Reply:
it already rolled back :) https://github.com/rectorphp/rector-src/blob/9ec3967a5bb51519dec381255b7c6c0659a41d6e/composer.json#L43, see PR:

- https://github.com/rectorphp/rector-src/pull/3335